src/python/pants/backend/python/rules/repl.py

```diff
@@ -1,9 +1,8 @@
 # Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
Same changes as the original implementation. This means that `PATH` will be determined by `--pex-executable-search-paths`.